jdaviz/core/template_mixin.py

```diff
@@ -3135,8 +3135,10 @@ def float_precision(column, item):
                     # stored in astropy table as a float so we can also store nans,
                     # but should display in the UI without any decimals
                     return int(item)
-                elif column in ('pixel'):
+                elif column in ('pixel', ):
                     return f"{item:0.03f}"
+                elif column in ('sum', ):
+                    return f"{item:.3e}"
```

Comment:
I thought this would work but it does not affect the front-end somehow. Need advice from @kecnry .

![Screenshot 2023-10-24 133136](https://github.com/spacetelescope/jdaviz/assets/2090236/fe1d8b91-c1fe-42e9-a142-e62aeda0964c)

Reply:
is the json getting passed showing the string formatting you'd expect?

Reply:
How do I check?

Reply:
Debug statements in this method or looking at `table.items`

Reply:
```python
>>> plg = imviz.plugins["Aperture Photometry"]
>>> plg._obj.table.items
[{'id': 1,
  ...,
  'sum': '529.3 electron',
  ...}]
```

Maybe the fix should be here but I am afraid to touch it:

https://github.com/spacetelescope/jdaviz/blob/f5a2a46c2b65e81f0ef8a2e03465a1646427d487/jdaviz/core/template_mixin.py#L3162-L3168

Reply:
Okay, I touched it and I think I fixed it. Commit incoming.